### PR TITLE
kv/RocksDB: add perfcounter for submit_transaction_sync operation

### PR DESCRIPTION
**Pull request by tanghaodong25**
We need collect the sum of submit_transaction and submit_transaction_sync respectively.

Signed-off-by: Haodong Tang <haodong.tang@intel.com>

### CODE REVIEW

File: src/kv/RocksDBStore.cc [hunk 1]
```diff
@@ -297,7 +297,8 @@ int RocksDBStore::do_open(ostream &out, bool create_if_missing)
 
   PerfCountersBuilder plb(g_ceph_context, "rocksdb", l_rocksdb_first, l_rocksdb_last);
   plb.add_u64_counter(l_rocksdb_gets, "rocksdb_get", "Gets");
-  plb.add_u64_counter(l_rocksdb_txns, "rocksdb_transaction", "Transactions");
+  plb.add_u64_counter(l_rocksdb_txns, "rocksdb_submit_transaction", "Submit transactions");
+  plb.add_u64_counter(l_rocksdb_txns_sync, "rocksdb_submit_transaction_sync", "Submit transactions sync");
```
Comment (liewegas) [inline src/kv/RocksDBStore.cc]: while you are here, do you mind removing the "rocksdb_" prefix from all of these?  they are already inside the "rocksdb" perfcounter section so it is redundant.

**Comment by liewegas** [comment]
otherwise looks good to me!

**Comment by tanghaodong25** [comment]
@liewegas  updated. Thanks.

**Comment by yuriw** [comment]
http://pulpito.ceph.com/yuriw-2016-07-12_13:07:42-rados-wip-yuri-testing_2016_7_12-distro-basic-smithi

**Pull request merged**